packages/back-end/src/api/openapi/payload-schemas/PostSdkConnectionPayload.yaml

```diff
@@ -0,0 +1,38 @@
+type: object
+required:
+  - name
+  - languages
+  - environment
+properties:
+  name:
+    type: string
+  languages:
+    type: array
+    items:
+      type: string
```

Comment:
For all of these new endpoints, we should use singular `language` here, make it optional, and an enum of SDKLanguage options.  We deprecated the ability to have multiple languages and require a single language for all new SDK Connections going forward.